config.yml

```diff
@@ -1,52 +1,55 @@
-# Name of app
-app_name: Inferno
-
-# Base path of application
+# Base path of application. Inferno will serve pages at HOST/base_path
```

Comment:
1) Is there a variable for HOST? or definition of what is HOST?
2) Maybe we could follow FHIR url pattern: [HOST]/[base_path]